### PR TITLE
Avoid division by zero

### PR DESCRIPTION
**Pull request by iriberri**
Fixes https://github.com/CartoDB/cartodb/issues/6212

When the CartoDBfy time was equal to 0, the throughput metric was computed as NaN, which provoked some issues in metrics.

Now, if the time is equal to 0, the `cartodbfy_throughput` value is set to 0 as well.

Sample output from a log with `cartodbfy_time == 0`

````json
{  
  "user":"cdb",
  "state":"failure",
  "tables":1,
  "imported_tables":0,
  "failed_tables":1,
  "error_code":2004,
  "import_timestamp":"2015-12-10T16:33:48+00:00",
  "queue_server":"cdb",
  "database_host":"localhost",
  "service_name":"public_url",
  "data_type":"file",
  "is_sync_import":false,
  "import_time":48.550614875,
  "file_stats":[  
    {  
      "type":".xlsx",
      "size":10856745,
      "file_rows":null,
      "imported_rows":86,
      "error_percent":null,
      "fallback_executed":null
    }
  ],
  "resque_ppid":5854,
  "user_timeout":13981,
  "error_source":"user",
  "id":"X",
  "total_size":10856745,
  "cartodbfy_time":0.0,
  "import_throughput":223617.04435571271,
  "cartodbfy_throughtput":0,
  "cartodbfy_import_ratio":0.0,
  "extra_options":{  

  },
  "retrieved_items":0
}
````

### CODE REVIEW

**Comment by iriberri** [comment]
cc @rafatower

File: app/models/data_import.rb [hunk 1]
```diff
@@ -772,6 +772,7 @@ def notify(results)
     importer_stats_aggregator.update_counter('total_size', total_size)
 
     import_time = self.updated_at - self.created_at
+    cartodbfy_throughtput = self.cartodbfy_time == 0 ? 0 : (total_size / self.cartodbfy_time)
```
Comment (rafatower) [inline app/models/data_import.rb]: I'd write:
```ruby
cartodbfy_throughtput = (cartodbfy_time == 0.0 ? nil : (total_size / cartodbfy_time))
```

Comment (iriberri) [inline app/models/data_import.rb]: On it! I thought we agreed on using 0, I'll change it.

**Comment by iriberri** [comment]
````
{  
  "user":"cdb",
  "state":"failure",
  "tables":1,
  "imported_tables":0,
  "failed_tables":1,
  "error_code":2004,
  "import_timestamp":"2015-12-11T11:17:31+00:00",
  "queue_server":"cdb",
  "database_host":"localhost",
  "service_name":"public_url",
  "data_type":"file",
  "is_sync_import":false,
  "import_time":48.565422651,
  "file_stats":[  
    {  
      "type":".xlsx",
      "size":10856745,
      "file_rows":null,
      "imported_rows":86,
      "error_percent":null,
      "fallback_executed":null
    }
  ],
  "resque_ppid":9104,
  "user_timeout":13981,
  "error_source":"user",
  "id":"X",
  "total_size":10856745,
  "cartodbfy_time":0.0,
  "import_throughput":223548.86269637872,
  "cartodbfy_throughtput":null,
  "cartodbfy_import_ratio":0.0,
  "extra_options":{  

  },
  "retrieved_items":0
}
````

**Comment by rafatower** [comment]
:+1:

**Comment by rafatower** [comment]
this will almost automatically raise some errors to the surface btw

**Comment by iriberri** [comment]
Thanks, I'll wait again for the tests and merge. Let's see what this uncovers.

**Pull request merged**